templates/predict.ipynb

```diff
@@ -21,34 +21,27 @@
    ],
```

Comment:
<p>How long does this take for a single 2048x2048 FOV?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/cell_classification/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
On my notebook i7 11800H 8-core and SSD (3.5G/sec read): with multiprocessing=True 0.25 sec/FOV, multiprocessing=False 3.9sec/FOV. So this is blazing fast for machines with SSD, but probably really slow for old machines with HDD (e.g. 80-160mb/sec read).

Reply:
<p>Okay got it. I think setting this default closer to 20 or 40 FOVs would be better. I guess you could also look and see what the sampling distribution looks like. Your call</p>

Reply:
Good call, I'll look into it.